modules/standard/Buffers.chpl

```diff
@@ -361,19 +361,19 @@ module Buffers {
   }
 
   pragma "no doc"
-  proc buffer.flatten(range:buffer_range) {
+  proc buffer.flatten(range:buffer_range) throws {
   //proc buffer.flatten(range:buffer_range):bytes {
     var error:syserr = ENOERR;
     var ret:bytes = new bytes();
     ret = this.flatten(range,error);
-    if error then ioerror(error, "in buffer.flatten");
+    if error then try ioerror(error, "in buffer.flatten");
     return ret;
   }
 
   // TODO -- shouldn't have to write this this way!
   pragma "init copy fn"
   pragma "no doc"
-  proc chpl__initCopy(x: buffer) {
+  proc chpl__initCopy(x: buffer) throws {
```

Comment:
Take care, this is the first throwing initCopy that I know of. It would be OK with me to make this one try!.

Reply:
`try!` seems prudent here, thanks.